packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -279,7 +279,11 @@ abstract class Domain {
     }).then<Object?>((Object? result) {
       daemon.connection.sendResponse(id, _toJsonable(result));
       return null;
-    }).catchError((Object error, StackTrace stackTrace) {
+    })
+      // TODO(srawlins): Fix this static issue,
+      // https://github.com/flutter/flutter/issues/105750.
+      // ignore: body_might_complete_normally_catch_error
+      .catchError((Object error, StackTrace stackTrace) {
```

Comment:
what does a real fix look like here?

Reply:
Maybe `return null` after `sendErrorResponse`? According to https://github.com/dart-lang/sdk/issues/49215#issuecomment-1151132073

Reply:
Should we just do that then?

Reply:
Done! I think this is valid.